### ISSUE TITLE
Cannot change locale?

### ISSUE THREAD

**Issue by pmoust** [Shippable/support#994]
```
/home/shippable/build/build.sh: line 89: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
```

When passing 
```
<snip>
install:
  - LC_ALL="en_US.UTF-8" && whatever
<snip>
```

**Comment by manishas** [comment]
@pmoust we will take a look at this... is your build failing because of this?

**Comment by pmoust** [comment]
@manishas i am forced to start daemons with LC_ALL="C" in a one liner with the service start cmd to make it work

**Comment by rageshkrishna** [comment]
@pmoust Which docker image are you using? I did a quick check with shippable/minv2, shippableimages/ubuntu1404_python and ubuntu1204_python and they all seem to have no issues switching to en_US.UTF-8. Thanks.

**Comment by pmoust** [comment]
shippable/minv2

cannot reproduce this any more.

**Comment by manonthemat** [comment]
I actually ran into the same issue.
I've used the official *node* docker hub image. Everything was working fine. Building the image, running the tests and pushing it into my private docker hub repository. Now when I want to add the deploy step to elastic beanstalk as described in several blog posts and the documentation, that's when this error occurred. I'm running Ubuntu 14.04 on the dedicated host.
